components/ApiKeyRow.js

```diff
@@ -0,0 +1,44 @@
+import { Key } from '@styled-icons/fa-solid/Key'
+import { Component } from 'react'
+import { Button, ListGroup } from 'react-bootstrap'
+
+/**
+ * Shows an individual API key with management features (create, delete). Delete
+ * is only permitted for admin users.
+ */
+export default class ApiKeyRow extends Component {
+  _deleteKey = () => this.props.deleteKey(this.props.apiKey)
+
+  _viewApiKey = () => {
+    const { apiKey } = this.props
+    window.prompt('Copy and paste the API key to use in requests', apiKey.value)
+  }
+
+  render () {
+    const { apiKey, isAdmin } = this.props
+    return (
+      <ListGroup.Item as='li'>
+        <Key size={20} style={{marginRight: 10}} />
+        {apiKey.keyId}{' '}
```

Comment:
I'm not sure it's useful to show this keyId to the user. Can we show just the apiKey value?

Reply:
I'm not sure there's space to show the entire apiKey value. Do you think we should hide the overflow or wrap it? I also thought that it might not always be a good idea to have these on full display on the screen (e.g., someone could take a screenshot during a zoom call and steal all the values).

Reply:
I can just see someone getting confused between what the ApiKey ID is and what the value is. Maybe if we only showed the first few characters and then had a view button to show the entire thing that could work.

Reply:
Yea, I had some concerns about folks confusing the two strings as well. I like your take. Will implement.

Reply:
Are you intending to implement @evansiroky's suggestion in this PR?

Reply:
No, doing so would require a server change. We can continue the conversation in user testing discussions as mentioned on Teams.